### ISSUE TITLE
Undefined method `step_match' for class `Cucumber::Runtime::SupportCode' (NameError)

### ISSUE THREAD

**Issue by abinoam** [makandra/cucumber_priority#1]
This happened after this upgrade:
```bash
Installing spreewald 1.5.1 (was 1.2.6)
Installing cucumber 2.3.2 (was 1.3.20)
Installing cucumber-rails 1.4.3 (was 1.4.2)
Installing cucumber_priority 0.1.1
Installing gherkin 3.2.0 (was 2.12.2)
Installing cucumber-core 1.4.0
Installing cucumber-wire 0.0.1
```

(I may be wrong) But I think it could be related to this cucumber commit https://github.com/cucumber/cucumber-ruby/commit/7fa9033bface285bb3aa301df69b563ca445742d

More specifically this line https://github.com/cucumber/cucumber-ruby/commit/7fa9033bface285bb3aa301df69b563ca445742d#diff-77cbba38b55a36e1e77aa3ebc64b03f3L151

Sorry, but I don't have expertise enough to dig deeper.

Backtrace bellow.

```bash
undefined method `step_match' for class `Cucumber::Runtime::SupportCode' (NameError)
.../gems/activesupport-4.2.5.1/lib/active_support/core_ext/module/aliasing.rb:32:in `alias_method'
.../gems/activesupport-4.2.5.1/lib/active_support/core_ext/module/aliasing.rb:32:in `alias_method_chain'
.../gems/cucumber_priority-0.1.1/lib/cucumber_priority/support_code_ext.rb:28:in `<class:SupportCode>'
.../gems/cucumber_priority-0.1.1/lib/cucumber_priority/support_code_ext.rb:3:in `<class:Runtime>'
.../gems/cucumber_priority-0.1.1/lib/cucumber_priority/support_code_ext.rb:2:in `<module:Cucumber>'
.../gems/cucumber_priority-0.1.1/lib/cucumber_priority/support_code_ext.rb:1:in `<top (required)>'
```

**Comment by henning-koch** [comment]
This should be fixed in version 0.1.2.

**Comment by abinoam** [comment]
Thanks for your feedback @henning-koch ! And thanks for the gems (spreewald also).
